### PR TITLE
fix mmcv-full

### PR DESCRIPTION
**Pull request by sanbuphy**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

renamed the package names mmcv-full to mmcv

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/simple_copy_paste/mask-rcnn_r50_fpn_rpn-2conv_4conv1fc_syncbn-all_32xb2-ssj-270k_coco.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 norm_cfg = dict(type='SyncBN', requires_grad=True)
 # Use MMSyncBN that handles empty tensor in head. It can be changed to
 # SyncBN after https://github.com/pytorch/pytorch/issues/36530 is fixed
-# Requires MMCV-full after  https://github.com/open-mmlab/mmcv/pull/1205.
+# Requires MMCV after  https://github.com/open-mmlab/mmcv/pull/1205.
```
Comment (ZwwWayne) [inline configs/simple_copy_paste/mask-rcnn_r50_fpn_rpn-2conv_4conv1fc_syncbn-all_32xb2-ssj-270k_coco.py:14]: we can delete this line of comments because mmdet 3.x relies on mmcv 2.x, which must have support it.

**Review by ZwwWayne — APPROVED** [review]
